mmv1/third_party/terraform/tests/resource_os_config_os_policy_assignment_test.go.erb

```diff
@@ -0,0 +1,147 @@
+<% autogen_exception -%>
+
+package google
+
+import (
+	"context"
+	"fmt"
+	dcl "github.com/GoogleCloudPlatform/declarative-resource-client-library/dcl"
+	osconfig "github.com/GoogleCloudPlatform/declarative-resource-client-library/services/google/osconfig<%= dcl_version(version) -%>"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"strings"
+	"testing"
+)
+
+func TestAccOsConfigOsPolicyAssignment_PercentOsPolicyAssignment(t *testing.T) {
+	// Tests fail due to long rollout times
```

Comment:
What about the rollout times makes them fail in VCR but not CI?

Reply:
They fail in CI too, been failing for months. I can add skip entirely as this resource is fairly untestable in our CI

Reply:
Thanks! Makes sense.